plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -2416,19 +2416,27 @@ public static function update_auth_option( string $access_token, string $access_
 	}
 
 	/**
-	 * Get base URL for plugin auto installer.
+	 * Get WooCommerce.com base URL.
 	 *
 	 * @return string
 	 */
-	public static function get_install_base_url() {
+	public static function get_woocommerce_com_base_url() {
 		/**
 		 * Filter the base URL used to install the Woo hosted plugins.
 		 *
 		 * @since 8.7.0
 		 */
-		$woo_com_base_url = apply_filters( 'woo_com_base_url', 'https://woocommerce.com/' );
+		return trailingslashit( apply_filters( 'woo_com_base_url', 'https://woocommerce.com/' ) );
```

Comment:
nitpick: Since we always refer to WordPress.com and WooCommerce.com as WPCOM and WCCOM respectively, what do you think about naming this filter `wccom_base_url`?